planning/autoware_rtc_interface/src/rtc_interface.cpp

```diff
@@ -317,6 +317,29 @@ bool RTCInterface::isActivated(const UUID & uuid) const
   return false;
 }
 
+bool RTCInterface::isForceActivated(const UUID & uuid) const
+{
+  std::lock_guard<std::mutex> lock(mutex_);
+  const auto itr = std::find_if(
+    registered_status_.statuses.begin(), registered_status_.statuses.end(),
+    [uuid](auto & s) { return s.uuid == uuid; });
+
+  if (itr != registered_status_.statuses.end()) {
+    if (itr->state.type > State::RUNNING) {
```

Comment:
[imo] it's better not to create implicit rule.
```suggestion
    if (itr->state.type == State::SUCCEEDED || itr->state.type == State::FAILED) {
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/8094/commits/303ff4cf79e8033c13890989a4ce43115bf6bebb